sdk/communication/azure-communication-email/azure/communication/email/aio/_email_client_async.py

```diff
@@ -77,16 +81,86 @@ def from_connection_string(
     @distributed_trace_async
     async def send(
         self,
-        email_message: EmailMessage,
+        email_message: JSON,
```

Comment:
In the generated code we support `Union[JSON, IO]` as input - is there any reason you removed the IO support from the convenience layer?

Reply:
No specific reason. I'll add this in the wrapper.